cli/kubernetes/platform/generic/generic.go

```diff
@@ -12,8 +12,17 @@ type Generic struct {
 	InternalIPs, ExternalIP []string
 }
 
-func (k *Generic) HasLoadBalancer() bool {
-	return true
+func (k *Generic) HasLoadBalancer(kube *kubernetes.Clientset) (bool, error) {
```

Comment:
The `HasLoadBalancer` method is supposed to return true when the cluster is deployed on a cloud (or other) where an external IP is automatically assigned (by an external load balancer). I couldn't find a way to introspect the cluster, I'm not sure that information is available. So I opted for hardcoded value on the platform. Default is `true` and we set it to `false` on the platforms we know there is no automatic IP assignment (e.g. kind).

Reply:
As we discussed, we cannot hardcode the value as local kube platforms have addons which enable the ability of the local kube platforms to provision IP address.